ols/app/models/config.py

```diff
@@ -823,7 +823,7 @@ class Config(BaseModel):
 
     llm_providers: Optional[LLMProviders] = None
     ols_config: Optional[OLSConfig] = None
-    dev_config: Optional[DevConfig] = None
+    dev_config: Optional[DevConfig] = DevConfig()
```

Comment:
This is the main change, the rest is just replacing how the config is accessed - not from the `config` module, but from the `config` variable holding the `Config` class.

Reply:
is it optional then?

Reply:
From the `Config` perspective, yes. It is optional, we just init `DevConfig` with default values instead of `None`. Which is the same as "before", where the `DevConfig` was initialized under global.

Reply:
I mean, if it's still Optional from type system perspective.

Reply:
anyway, it's nitpick